### DISCUSSION TITLE
Docs are 💎

### DISCUSSION THREAD

**Discussion by fedeolto** [tailwindlabs/tailwindcss#4643] [Kind words]
Just to say that I'm very impressed by the "efficiency" of the Tailwind Css docs. 
I've never learned so fast in so small time, docs are short but complete and this inspires trust. 
I think that this is a big humanistic achievement and I'm curious about the process you adopt for writing and refining the docs, I think that it would be great if others could adopt it

**Comment by VicGUTT** [comment]
I wholeheartedly agree ! 👌
